internal/resources/flush_bucket_schema.go

```diff
@@ -0,0 +1,16 @@
+package resources
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+)
+
+func FlushBucketSchema() schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"organization_id": stringAttribute([]string{required, requiresReplace}),
+			"project_id":      stringAttribute([]string{required, requiresReplace}),
+			"cluster_id":      stringAttribute([]string{required, requiresReplace}),
```

Comment:
IMO `requiresReplace` does not apply to this  resource.  Terraform will try to flush a bucket in a new environment.